examples/css/style.css

```diff
@@ -0,0 +1,28 @@
+#entry1 {
+    background-image: -gtk-gradient (linear,
+                                     0 0, 1 0,
+                                     color-stop(0, #f00),
+                                     color-stop(1, #0f0));
+    color: blue;
+    font-weight: bold;
+}
+
+button {
+    /* If we don't put it, the yellow background won't be visible */
+    background-image: none;
+}
+#label1:hover {
+    transition: 500ms;
+    color: red;
+    background-color: yellow;
+}
+
+combobox button.combo box {
+    padding: 5px;
+}
+combobox box arrow {
+    -gtk-icon-source: none;
+    border-left: 5px solid transparent;
+    border-right: 5px solid transparent;
+    border-top: 5px solid black;
+}
```

Comment:
Better to have a newline at the end

Reply:
Done.
Also added the setting to my editor, even though I never understood what difference it makes.

Reply:
It make diffs more useful to read later at least :)